app/common/renderer/components/Session/AttachToSession.jsx

```diff
@@ -1,45 +1,43 @@
 import {ReloadOutlined} from '@ant-design/icons';
-import {Button, Card, Col, Form, Row, Select} from 'antd';
+import {Button, Card, Col, Form, Row, Select, Tooltip} from 'antd';
+import _ from 'lodash';
 import React from 'react';
 
 import {ServerTypes} from '../../actions/Session';
 import SessionStyles from './Session.module.css';
 
-const formatCaps = (caps) => {
-  let importantCaps = [caps.app, caps.platformName, caps.deviceName];
-  if (caps.automationName) {
-    importantCaps.push(caps.automationName);
+const formatCaps = (initialCaps, serverType) => {
+  let returnedCaps = [];
+  const caps =
+    serverType === ServerTypes.lambdatest && 'capabilities' in initialCaps
+      ? initialCaps.capabilities
+      : initialCaps;
+  // add sessionName (BrowserStack and other cloud providers only)
+  returnedCaps.push(caps.sessionName);
```

Comment:
I assume it should only be pushed if not nil

Reply:
Since some of the other caps may also end up being nil, I figured it's safer to just filter them out only once the whole list is assembled (on line 35)